scissors/src/main/java/com/lyft/android/scissors/CropViewExtensions.java

```diff
@@ -73,6 +79,17 @@ public LoadRequest using(@Nullable BitmapLoader bitmapLoader) {
         }
 
         /**
+         * Load a {@link Bitmap} using the {@link BitmapLoader} specified by {@code bitmapLoaderReference}, you must call {@link LoadRequest#load(Object)} afterwards.
+         *
+         * @param bitmapLoaderReference a reference to the {@link BitmapLoader} to use
+         * @return current request for chaining, you should call {@link #load(Object)} afterwards.
+         */
+        public LoadRequest using(@CropView.Extensions.ExtensionBitmapLoader int bitmapLoaderReference) {
+            this.bitmapLoaderReference = bitmapLoaderReference;
+            return this;
```

Comment:
We would need to check for invalid values here to eagerly raise the error, we'll do it in a separate PR :)